import-scripts/fetch-dmp-data-for-import.sh

```diff
@@ -394,6 +418,14 @@ MY_FLOCK_FILEPATH="/data/portal-cron/cron-lock/fetch-dmp-data-for-import.lock"
                 cd $MSK_ACCESS_PRIVATE_DATA_HOME ; $GIT_BINARY add ./* ; $GIT_BINARY commit -m "Latest ACCESS dataset"
             fi
         fi
+
+        # Upload ACCESS sample list to S3 for CDM use
```

Comment:
if upload fails, will previous days file still be available? Or does it get corrupted